appliances/cisco-iosvl2.gns3a

```diff
@@ -32,21 +39,27 @@
         },
         {
             "filename": "vios_l2-adventerprisek9-m.vmdk.SSA.152-4.0.55.E",
-            "version": "15.2.4055",
+            "version": "15.2(4055)E",
```

Comment:
It should be 15.2(4.0.55)E to match with what we see when doing a "show version" in IOS.